styles/style.css

```diff
@@ -0,0 +1,219 @@
+* {
+  box-sizing: border-box;
+}
+
+.header-img {
+  background-image: url(../images/header-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 20em;
+}
+
+.logo {
+  max-width: 40%;
+  margin-left: 1em;
+  margin-top: 1em;
+}
+
+.tagline {
+  color: 	#81B642;
+  font-size: 2vw;
+  font-style: italic;
+  font-weight: 200;
+  margin: 0;
+  margin-left: 1em;
+}
+
+.cart-total {
+  color: white;
+  position: absolute;
+  left: 80%;
+  top: 3em;
+  width: 12em;
+}
+
+.cart-icon {
+  max-width: 1em;
+}
+
+.checkout-btn {
+  background-color: #81B642;
+  border: none;
+  color: white;
+  padding: 5px 18px;
```

Comment:
Keep to REMS/EMS for this, you'll want to be consistent

Reply:
Done.